components/Form/EmailInput/EmailInput.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
```

Comment:
Sorry to be so picky but we can simplify a lot these components:

```
import { TextInput } from '..';

const EmailInput = (props) => (
    <TextInput {...props} type="email"></TextInput>
);

export default EmailInput;
```

the wrapper `<div className={cx('govuk-form-group')}>` is defined in `TextInput`, if we keep both it will render:
```
<div className="govuk-form-group">
  <div className="govuk-form-group">
```

Same thing for the `propTypes` that are already checked into the `TextInput`.

Reply:
Done, was curious about the type checking for prop types if it needed to be done in both components, thanks for the clarification.

Reply:
if you check the warning in the console you will have this:
```
Failed prop type: The prop `label` is marked as required in `TextInput`, but its value is `undefined`.
    in TextInput (at PhoneInput.jsx:8)
    in div (at PhoneInput.jsx:7)
    in PhoneInput (at DynamicInput.jsx:29)
    in DynamicInput (at DynamicStep.jsx:17)
    in div (at DynamicStep.jsx:14)
    in form (at DynamicStep.jsx:13)
    in DynamicStep (at FormWizard.jsx:93)
    in fieldset (at FormWizard.jsx:60)
    in div (at FormWizard.jsx:53)
    in FormWizard (at [stepId].jsx:13)
    in ChildReferral (at _app.jsx:28)
    in main (at Layout/index.jsx:14)
    in div (at Layout/index.jsx:12)
    in Layout (at _app.jsx:26)
    in MyApp
    in ErrorBoundary (created by ReactDevOverlay)
    in ReactDevOverlay (created by Container)
    in Container (created by AppContainer)
    in AppContainer
    in Root
```

Where you can see the stack error, as we are passing all the props directly from the `PhoneInput` to the `TextInput` one check is enough.
Let me know if you have any more question :)